japicmp/src/main/java/japicmp/compat/CompatibilityChanges.java

```diff
@@ -265,6 +265,7 @@ private void checkIfConstructorsHaveChangedIncompatible(JApiClass jApiClass, Map
 				}
 			}
 			checkIfAnnotationDeprecatedAdded(constructor);
+			checkIfVarargsChanged(constructor);
```

Comment:
Have also added this check for constructors, though the names `METHOD_NOW_VARARGS` and `METHOD_NO_LONGER_VARARGS` might be a bit misleading then. Should I rename then?